cg/meta/report/presenter.py

```diff
@@ -0,0 +1,66 @@
+"""Handles presentation of different data types"""
+from datetime import datetime
+
+
+class Presenter:
+    """Contains methods for presentation of different data types"""
+
+    @staticmethod
+    def process_float_string(float_str: str, precision: int) -> str:
+        """Make a float value presentable for the delivery report."""
+        if float_str:
+            presentable_value = str(round(float(float_str), precision))
+        else:
+            presentable_value = 'N/A'
```

Comment:
I know ternary operators are to be avoided and the best use case they have is to write assignments like this into one line. Is it more readable to spell it out like it is now? probably.

Reply:
Let's try the ternary one :)